templates/webtest/demo_viewers/static_example.html

```diff
@@ -0,0 +1,177 @@
+<!doctype html>
+<!-- See discussion at https://www.openmicroscopy.org/community/viewtopic.php?f=6&t=8058&p=17114 -->
+<html>
+<head>
+
+    <link href="https://cowfish.openmicroscopy.org/webmerge/static/omeroweb.viewer.min.css" type="text/css" rel="stylesheet"></link>
+    <script src="https://cowfish.openmicroscopy.org/webmerge/static/omeroweb.viewer.min.js" type="text/javascript"></script>
+
+    <!--
+    <script type="text/javascript" src="http://localhost:4080/static/3rdparty/jquery-1.11.1.js"></script>
+    <script type="text/javascript" src="http://localhost:4080/static/webgateway/js/ome.csrf.js?_5.3.0-m3-ice35-SNAPSHOT"></script>
```

Comment:
The same here. If you want to include direct links I would suggest to use version number matching recent release and make a comment is up to developer to maintain all these links

I am guessing we should also remove ``localhost:4080`` and replace it with something like ``omero.host`` (I am also interested what other people think about pointing to nightshade?)